guides/v2.3/graphql/queries/cart.md

```diff
@@ -523,6 +523,183 @@ The `3T1free` rule is applied first, and Magento returns the price of a single s
 }
 ```
 
+### Tier price example
+
+In the following example, tier prices has been established for product `24-UG01` and `24-UG05`, as shown in the following table:
+
+Product | Quantity | Fixed/Discount | Amount
+--- | --- | --- | --- | 
+24-UG01 | 5 | Discount | 5%
+24-UG01 | 10 | Discount | 10%
+24-UG01 | 15 | Discount | 15%
+24-UG05 | 5 | Fixed | $16
+24-UG05 | 10 | Fixed | $11
+
+The cart in the example contains 12 units of `24-UG05` and 8 units of `24-UG-01`, so the price of `24-UG05` is $11, and the price of `24-UG01` is $18.05 (10% off).
```

Comment:
are units and quantity the same? I count 15 UG05 and 30 UG01.

Reply:
Those are the tier definitions--they are not to be added. Line 598 (`"quantity": 12,`) shows the quantity of 24-UG05, while line 633 shows that the quantity for 24-UG01 is 8. I did make an error in saying that $18.05 reflects a 10% discount. It's actually 5%.